floodviz/static/js/map.js

```diff
@@ -61,10 +63,10 @@
 				.attr("transform", function (d) {
 					return "translate(" + projection(d.geometry.coordinates) + ")";
 				})
-				.attr("id", function(d) {
-					if (classname === 'gage-point') {
-						return 'map' + d.properties.id;
-					} else { return '';}
+				.attr("id", function (d) {
+					if (property_for_id && d.properties[property_for_id]) {
+						return 'map' + d.properties[property_for_id];
+					}
```

Comment:
I really think it depends on the usage. In this case I think the if else is fine. But I have cases where there are a series of variables that need to have values. Rather than have a series of if else clauses, the code looks a lot cleaner if you use the ternary operator. As with most things, if you get use to it, it looks more natural.